archeryutils/classifications/agb_field_classifications.py

```diff
@@ -332,11 +258,6 @@ def agb_field_classification_scores(
     # Get scores required on this round for each classification
     class_scores = group_data["class_scores"]
 
-    # Make sure that hc.eq.score_for_round did not return array to satisfy mypy
-    if any(isinstance(x, np.ndarray) for x in class_scores):
-        raise TypeError(
-            "score_for_round is attempting to return an array when float expected."
-        )
     # Score threshold should be int (score_for_round called with round=True)
     # Enforce this for better code and to satisfy mypy
     int_class_scores = [int(x) for x in class_scores]
```

Comment:
I _think_ these lines are now superfluous here, given `score_for_round` isn't called and `GroupData` now carries the information that the scores returned are `int`s?

Reply:
Yes, that does sound right. I think I hesitated here because the outdoor/new indoor ones are generated via score_for_round. Which incidentally I tried to type accounting for the round=True with overloads but it defeated me!